### DISCUSSION TITLE
Why is there no support for Playwright Assertions?

### DISCUSSION THREAD

**Discussion by ATheCoder** [artilleryio/artillery#2451] [Artillery]
Basically what the title says. If we omit playwright assertions how do we know that things work properly under load?

**Comment by hassy** [comment]
hi @ATheCoder 👋 there's an example using `expect()` calls here: 

https://github.com/artilleryio/artillery/blob/main/examples/browser-playwright-reuse-authentication/flow.js

If any of those fail Artillery will report a failed VU with the `vusers.failed` metric.

From then on you'd look at monitoring & tracing data - either Artillery's traces (https://www.artillery.io/blog/end-to-end-tracing-for-playwright-tests) or traces produced by the app you're testing.

**Comment by hassy** [comment]
(moving to Discussions)
